src/chains/details/polygon.ts

```diff
@@ -0,0 +1,45 @@
+import { nativeCurrencyTemplate } from '../../common/consts/tokens'
+import { ChainInfo } from '../types'
+import { RAW_CHAINS_FILES_PATH } from '../const'
+
+const polygonLogo = `${RAW_CHAINS_FILES_PATH}/polygon-logo.svg`
+
+// See https://github.com/wevm/viem/blob/main/src/chains/definitions/polygon.ts
+export const polygon: ChainInfo = {
+  id: 137,
+  label: 'Polygon',
+  logo: { light: polygonLogo, dark: polygonLogo },
+  nativeCurrency: {
+    ...nativeCurrencyTemplate,
+    chainId: 137,
+    name: 'POL',
+    symbol: 'POL',
+    logoUrl: `${RAW_CHAINS_FILES_PATH}/polygon-logo.svg`,
```

Comment:
Is the native currency logo the same as the chain logo?

In this case:

```suggestion
    logoUrl: polygonLogo,
```

Reply:
Maybe, maybe not. Is possible we will want to refine these things later. For now I don't want to focus on this. 
We can add the 0xeeee address in the token-list project if we want to differenciate